guides/v8-snapshots.md

```diff
@@ -33,6 +33,14 @@ This cache should be maintained and updated over time. Rather than having this m
 
 ## Troubleshooting
 
+**Local Development**
+
+If you're running into problems locally, either with generating the snapshot or at runtime, a good first step is to clean everything and start from scratch. This command will accomplish that (note that it will delete any new unstaged files, so if you want to keep them, either stash them or stage them):
+
+```
+git clean -fxd && yarn
```

Comment:
any idea if we have this referenced for any of the sql lite mismatch issues as well?

Reply:
I can add that there. That's a good idea.